csharp_dotnetcore/9.Message-Routing/MessageRoutingBot.csproj

```diff
@@ -0,0 +1,110 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <CodeAnalysisRuleSet>MessageRoutingBot.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="CogSvcModels\**" />
+    <Compile Remove="DeploymentScripts\data\**" />
+    <Compile Remove="DeploymentScripts\Json\**" />
+    <Compile Remove="MainDialog\**" />
+    <Content Remove="CogSvcModels\**" />
+    <Content Remove="DeploymentScripts\data\**" />
+    <Content Remove="DeploymentScripts\Json\**" />
+    <Content Remove="MainDialog\**" />
+    <EmbeddedResource Remove="CogSvcModels\**" />
+    <EmbeddedResource Remove="DeploymentScripts\data\**" />
+    <EmbeddedResource Remove="DeploymentScripts\Json\**" />
+    <EmbeddedResource Remove="MainDialog\**" />
+    <None Remove="CogSvcModels\**" />
+    <None Remove="DeploymentScripts\data\**" />
+    <None Remove="DeploymentScripts\Json\**" />
+    <None Remove="MainDialog\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="Dialogs\TestCancelDialog.cs" />
+    <Compile Remove="Dialogs\TestDialog.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.5" />
+    <PackageReference Include="Microsoft.Azure.CognitiveServices.ContentModerator" Version="0.12.1-preview" />
+    <PackageReference Include="Microsoft.Azure.CognitiveServices.Language" Version="1.0.1-preview" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.Azure" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Builder.TemplateManager" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Configuration" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Connector" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Bot.Schema" Version="4.0.0.38762" />
+    <PackageReference Include="Microsoft.Graph" Version="1.10.0" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta008">
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
Missing the Async Analyzers here as well.

Reply:
I still don't see the async usage analyzer in here.. 

---
In reply to: [216445553](https://github.com/Microsoft/BotBuilder-Samples/pull/349#discussion_r216445553) [](ancestors = 216445553)